workers/unity/Packages/com.improbable.gdk.core/CoreSystemHelper.cs

```diff
@@ -14,8 +14,6 @@ public static void AddSystems(World world)
             world.GetOrCreateManager<WorldCommandsCleanSystem>();
```

Comment:
Off-topic: Is this file even used still at this point?

Reply:
the CoreSystemHelper? That's called from SystemConfig in playground and it is what adds all the systems to a World.